tests/integration-tests/CMakeLists.txt

```diff
@@ -55,10 +56,13 @@ target_link_libraries(
   mir-test-doubles-static
 
   mircommon
+  miral
+  miral-internal
```

Comment:
I wonder if this would be better as a separate, more focused, test executable? (Instead of bundling a lot of dependencies into an existing one)

Reply:
Yes